components/ImageDelete.js

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+import Button from "./Button";
+import Image from "next/image";
+import CheckUserExistence from "@/utils/CheckUserExistence";
+
+const StyledImageWrapper = styled.div`
+  width: 100%;
+  height: 300px;
+  position: relative;
+  overflow: hidden;
+`;
+
+export default function ImageDelete({ onClose, userId, userImage }) {
+  async function handleSubmitDelete() {
+    //delete user image in database
+    const userData = await CheckUserExistence({ userId });
+    const user_Id = userData._id;
+    try {
+      const newImage = { image: null };
```

Comment:
Setting `image: null` breaks the schema because `height`, `width`, and `url` are required. Instead of removing the entire field, you should reset its values to defaults.

```suggestion
      const newImage = image: {
          height: "",
          width: "",
          url: "/asset/user.png",
        }
```